packages/@okta/i18n/src/properties/login.properties

```diff
@@ -803,7 +803,7 @@ oie.okta_verify.totp.title = Enter a code
 oie.okta_verify.totp.enterCodeText = Enter code from Okta Verify app
 oie.okta_verify.push.title = Get a push notification
 oie.okta_verify.push.sent = Push notification sent
-oie.okta_verify.push.send = Send push notification
```

Comment:
I vaguely remember we auto push by default.. but i can't remember if that changed. Is there no case where we need `Send push notification`?

Reply:
If we don't need `Send push notification` does it make sense to change the key to `oie.okta_verify.push.resend`

Reply:
@gowthamidommety-okta  I can update the key name. Currently I only see this key used in one place https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/views/ov/ChallengeOktaVerifyResendPushView.js#L15

Reply:
got it. If you can change the key name / add a new key that would be great :)